roles/redis_exporter/tasks/configure.yml

```
@@ -0,0 +1,37 @@
+---
+- name: Copy the redis_exporter systemd service file
+  ansible.builtin.template:
+    src: redis_exporter.service.j2
+    dest: /etc/systemd/system/redis_exporter.service
+    owner: root
+    group: root
+    mode: 0644
+  notify: restart redis_exporter
+
+- name: Create redis_exporter config directory
+  ansible.builtin.file:
+    path: "/etc/redis_exporter"
+    state: directory
+    owner: root
+    group: root
+    mode: u+rwX,g+rwX,o=rX
+
+- name: Copy the passwords file
+  ansible.builtin.template:
```

I'm thinking maybe we should do this with `copy`.

```suggestion
  ansible.builtin.copy:
    content: "{{ redis_exporter_passwords | to_json(indent=2, sort_keys=True) }}"
```

---

nice trick, thank you. Update pushed.